src/providers/data_provider/dp_targets.c

```diff
@@ -304,16 +239,6 @@ static errno_t dp_target_special(struct be_ctx *be_ctx,
         }
     }
 
-    if (target->target == DPT_SUDO) {
-        if (dp_target_sudo_enabled(be_ctx)) {
-            return EAGAIN;
```

Comment:
I'm not sure in what channel we had this discussion but we agreed with Fabiano and Jakub to remove it and have the same behaviour in both cases. We just need to make it clear in release notes.